dynamiqs/solvers/ode/fixed_solver.py

```diff
@@ -27,30 +27,30 @@ def run_autograd(self):
             step inside `solver` and the time step inside the iteration loop. A small
             error can thus buildup throughout the ODE integration. TODO Fix this.
         """
-        # assert that `t_save` values are multiples of `dt`
+        # assert that `t_stop` values are multiples of `dt`
         if not torch.allclose(
-            torch.round(self.t_save / self.dt), self.t_save / self.dt
+            torch.round(self.t_stop / self.dt), self.t_stop / self.dt
         ):
             raise ValueError(
-                'For fixed time step solvers, every value of `t_save` must be a'
+                'For fixed time step solvers, every value of `t_stop` must be a'
```

Comment:
It's a bit annoying to raise an error with an argument that the user does not know about. I think we should keep `t_save` here, and ideally, also check separately for `t_meas` and raise a similar error. Or otherwise we just say `t_save` and `t_meas` in the same error message, specifying that `t_meas` is only for the SME.

Reply:
I'm being a bit picky here, but I think this is an error message that many users will see at least once.